subsys/bluetooth/nrf_bt_adv/providers/Kconfig.swift_pair

```
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_BT_ADV_PROV_SWIFT_PAIR
+	bool "Microsoft Swift Pair"
+	help
+	  Adds Microsoft Swfit Pair manufacturer data to advertising data if
+	  used Bluetooth local identity has no bonds and advertising has not
+	  entered grace period.
+
+if NRF_BT_ADV_PROV_SWIFT_PAIR
+
+config NRF_BT_ADV_PROV_SWIFT_PAIR_GRACE_PERIOD
+	int "Swift Pair grace period [s]"
```

I suggest using the official terminology "cool-down" as used in [Microsoft's guideline for Swift Pair].

```suggestion
config NRF_BT_ADV_PROV_SWIFT_PAIR_COOL_DOWN_DURATION
	int "Swift Pair cool down duration [s]"
```

[Microsoft's guideline for Swift Pair]: https://docs.microsoft.com/en-us/windows-hardware/design/component-guidelines/bluetooth-swift-pair

---

Ok, changed. Also introduced range from 30 to 600 seconds to ensure that value is valid (I think it's reasonable to assume that the cool down phase would not be longer than 10 minutes too)